packages/open-next/src/wrappers/cloudflare.ts

```diff
@@ -12,8 +12,16 @@ const handler: WrapperHandler<
 > =
   async (handler, converter) =>
   async (event: Request, env: Record<string, string>): Promise<Response> => {
-    //@ts-expect-error - process is not defined in cloudflare workers
-    globalThis.process = { env };
+    globalThis.process = process;
```

Comment:
Where is the `process` variable coming from? Is that specific to cloudflare?

Reply:
I added this as a js banner in the esbuild that i did for the cloudflare worker.
Now that i think about it we should probably do this esbuild part inside of OpenNext as well. We'd get a single file out of it which is easier for cloudflare worker.

I'll publish a sample repo later with SST v3 with different way to deploy OpenNext on different target.
Right now i have EC2, ECS and lambda with either cloudfront or cloudflare in front of it ( and with cloudflare the middleware runs in the worker )
Maybe that's something we could put in the new org as well